script.js

```diff
@@ -1,4 +1,117 @@
+// Define current player
+var currentPlayer = 1;
+
+// Function for output
+var myOutputValue = "";
```

Comment:
I strongly suggest keeping `myOutputValue` inside the `main` function, because it is the only function that `return`s the string back to the grey output box for rendering. You can view my comments to [Yuting's pull request](https://github.com/rocketacademy/basics-beat-that/pull/226#pullrequestreview-793589735) further on why, and how to achieve it.